### PR TITLE
Getting rid of developmental play page

### PR DESCRIPTION
**Pull request by tsevasa**
Previous dev.html is now play.html. Now there is no more /play/devversion.

### CODE REVIEW

File: src/client/views/play.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <!-- Our in-document javascript is injected here before the server sends this html out -->
+        <!-- Our in-document javascript is injected here before we send out the file required to monitor the loading animation -->
```
Comment (Naviary2) [inline src/client/views/play.html]: The in document javascript **is** the file required to monitor the losing animation.

But it also is in charge of loading our audio.

File: src/client/views/play.html [hunk 2]
```diff
@@ -10,25 +10,161 @@
...
+    <!-- The onload event listener to call main.start() is now within the htmlscript! -->
     <body>
         <header>
             <nav>
                 <a href="/">
                     <p>Home</p>
                 </a>
                 <a href="/play">
-                    <p class="currPage">Play</p>
+                    <p>Play</p>
```
Comment (Naviary2) [inline src/client/views/play.html]: This line can be reverted. Without it the "Play" link on the header isn't grey to indicate to you that is the page you're on.

File: src/client/views/play.html [hunk 3]
```diff
@@ -221,6 +357,8 @@ <h2>Board Editing</h2>
                                         <option value="Abundance">Abundance</option>
                                         <option value="Amazon Chandelier">Amazon Chandelier</option>
                                         <option value="Containment">Containment</option>
+                                        <!-- <option value="Standarch - 3 Check">Standarch - 3 Check</option>
+                                        <option value="CoaIP - 3 Check">Coaip - 3 Check</option> -->
```
Comment (Naviary2) [inline src/client/views/play.html:361]: 👍

File: src/client/views/play.html (hunk 2 above)
Comment (tsevasa) [reply src/client/views/play.html]: already did in my second commit

File: src/client/views/play.html (hunk 1 above)
Comment (tsevasa) [reply src/client/views/play.html]: already reverted in my second commit

Comment (Naviary2) [reply src/client/views/play.html]: Oh my bad 😖 I thought I was viewing all changes but it must have been a single commit!

**Pull request merged**